src/symbolDocumenter.ts

```diff
@@ -5,7 +5,7 @@ import * as path from 'path';
 import * as fs from 'fs';
 import { syntaxInfo } from './syntaxInfo';
 
-const commentLineRegex = /^;\s*(.*)$/
+const commentLineRegex = /^\s*;\s*(.*)$/
```

Comment:
At first glance, this seems fine, and I'm not sure why I didn't already support it. I'll play around with it and see if anything breaks in a weird way, but my first impression is this is an obvious improvement that makes a lot of sense.